test/AWS.jl

```diff
@@ -1,3 +1,7 @@
+function _get_formmated_date_now()
```

Comment:
Typo
```suggestion
function _get_formatted_date_now()
```
Alternatively:
```suggestion
function _now_formatted()
```